### PR TITLE
chore(release): allow prerelease

### PR DESCRIPTION
**Pull request by Haroenv**
The flow didn't change, except that now you can give a prerelease-style version and it will release on the right npm tag

### CODE REVIEW

**Comment by Haroenv** [comment]
I wrote it in JS because messing with regexes and stuff is really meh in bash, and doing scripts in JS is also quite meh

File: scripts/get-npm-tag.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+/* eslint-disable no-console */
+const { version } = require('../package.json');
+
+const getReleaseTag = _version => {
+  const [match = 'latest'] = _version.match(/[a-z]+/) || [];
+  return match;
+};
+
+// give this on to the main script
+console.log(getReleaseTag(version));
```
Comment (samouss) [inline scripts/get-npm-tag.js]: There is maybe a more "cleaner" way to output a value?

Comment (Haroenv) [reply scripts/get-npm-tag.js]: alternative is `process.stdout.write`. I'll change to that, only difference is that console.log will send a newline :

Comment (samouss) [reply scripts/get-npm-tag.js]: I'm fine with this implementation then, `console.log` actually calls `process.stdout.write` internally.

**Pull request merged**